### ISSUE TITLE
www.dplay.fi 

### ISSUE THREAD

**Issue by peace2000** [spaam/svtplay-dl#999]
Hello! If your goal is to support nordic video sites can you also add support for www.dplay.fi?

Best wishes from Finland.

**Comment by koskkim** [comment]
That would be very easily done by editing following rows at the **_dplay.py_** file:


**row 16:**
`country = {"sv": ".se", "da": ".dk", "no": ".no", "fi": ".fi"}
`

**row 20:**
`supported_domains = ['dplay.se', 'dplay.dk', "dplay.no", "dplay.fi"]
`

**row 63:**
`atch = re.search("(videos|videot|videoer)/(.*)$", parse.path)
`

That's it. Just tried it and it's working perfectly!

I'm a noob with GitHub, but somebody should make the change to the source code :)

**Comment by tomsku** [comment]
Yes please make those changes, Python was not installed to this machine and it's a pain to get everything working from source..
